api/src/services/records.js

```diff
@@ -167,6 +167,9 @@ const createByForm = (data, { locale }={}) => {
     throw new PublicError('Missing required field: message');
   }
 
+  // replace zero-width unicode characters
+  data.message = data.message.replace(/[\u200B-\u200D\uFEFF]/g, '');
```

Comment:
It actually is included. The little `-` in the middle there is defining a range which includes "u200C". Initially I didn't use a range because it's hard to see what it's doing. I've updated the code to make this more obvious.